### PR TITLE
test(contrib): add unit test for opensergo.go

### PR DESCRIPTION
**Pull request by zoujiejun**
#### Description (what this PR does / why we need it):

coverage: 88.0% of statements

#### Which issue(s) this PR fixes (resolves / be part of):

part of https://github.com/go-kratos/kratos/issues/2135

### CODE REVIEW

File: contrib/opensergo/opensergo_test.go [hunk 1]
```diff
@@ -0,0 +1,352 @@
...
+	pref "google.golang.org/protobuf/reflect/protoreflect"
+	"google.golang.org/protobuf/reflect/protoregistry"
+	"google.golang.org/protobuf/types/descriptorpb"
+	"io/ioutil"
+	"net"
+	"net/url"
+	"os"
+	"path/filepath"
+	"reflect"
+	"testing"
```
Comment (shenqidebaozi) [inline contrib/opensergo/opensergo_test.go:20]: import sort

Comment (zoujiejun) [reply contrib/opensergo/opensergo_test.go]: ok

**Comment by haiyux** [comment]
Please fix lint, use `make fix`

**Review by haiyux — APPROVED** [review]
LGTM

**Pull request merged**